openff/interchange/_tests/unit_tests/interop/openmm/_import/test_virtual_sites.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+from openff.toolkit import Quantity, Topology
+
+from openff.interchange import Interchange
+from openff.interchange._tests import MoleculeWithConformer
+from openff.interchange.components._packmol import solvate_topology
+from openff.interchange.drivers.openmm import _get_openmm_energies, get_openmm_energies
+
+
+class TestTIP4PVirtualSites:
+    def test_tip4p_openmm_xml(self, water_dimer):
+        """
+        Prepare a TIP4P water dimer with OpenMM's style of 4-site water.
+
+        Below is used as a guide
+        https://openmm.github.io/openmm-cookbook/latest/notebooks/tutorials/Histone_methyltransferase_simulation_with_a_multisite_water_model_TIP4P-Ew.html
+        """
+        pytest.importorskip("openmm")
+
+        import openmm.app
+
+        modeller = openmm.app.Modeller(
+            topology=water_dimer.to_openmm(),
+            positions=water_dimer.get_positions().to("nanometer").to_openmm(),
+        )
+
+        forcefield = openmm.app.ForceField("tip4pew.xml")
+
+        modeller.addExtraParticles(forcefield=forcefield)
+
+        system = forcefield.createSystem(
+            modeller.topology,
+            nonbondedMethod=openmm.app.PME,
+            nonbondedCutoff=1.0 * openmm.unit.nanometers,
+            constraints=openmm.app.HBonds,
+            rigidWater=True,
+            ewaldErrorTolerance=0.0005,
+        )
+
+        imported = Interchange.from_openmm(
+            topology=modeller.topology,
+            system=system,
+            positions=modeller.getPositions(),
+            box_vectors=modeller.getTopology().getPeriodicBoxVectors(),
+        )
+
+        get_openmm_energies(imported)
```

Comment:
This test could perhaps be improved by comparing the energies of `system` to `imported.to_openmm_system()` to ensure that they're within some bound? That would be a stopgap for the next test until `LocalCoordinatesSite` is supported.

Reply:
Good idea, 8100ad8514b471b26993f235306c51cab8879b9d